cli/storage_profile.c

```diff
@@ -466,6 +457,13 @@ hse_storage_profile(const char *path, bool quiet, bool verbose)
     for (i = 0; i < num_thread_counts; i++)
         scores[i] = NAN;
 
+    timer.it_value.tv_sec = test_time_secs;
+    rc = setitimer(ITIMER_REAL, &timer, NULL);
+    if (rc) {
+        fprintf(stderr, "Profiling failed: %s\n", strerror(rc));
+        goto err_exit;
+    }
+
```

Comment:
You should say why, "unable to arm internal timer" or some such...

Reply:
Sure, will add.